explorer-api/src/Explorer/Web/ClientTypes.hs

```diff
@@ -128,23 +127,17 @@ newtype CNetwork = CNetwork Text
 newtype CCoin = CCoin
     { unCCoin :: Integer
     } deriving (Show, Generic, Eq)
+      deriving newtype (Num, Ord)
```

Comment:
Add a `Num` instance removes some of the type safety the newtype wrapper was intended to add.

Reply:
Interesting. Could you give me an example?

I know you could now do constants like `let a = 0 :: CCoin`, but that seems fine to me. (Though I welcome debate. :-) )

You'd still be prevented from making mistakes this though:

```
let a = 5 :: Int
    b = 4 :: CCoin
in a + b
```

What am I missing? 🤔